service/worker/scheduler/spec_test.go

```diff
@@ -445,3 +451,30 @@ func (s *specSuite) TestSpecJitterSingleRun() {
 		time.Date(2022, 4, 7, 13, 57, 26, 927000000, time.UTC),
 	)
 }
+
+func (s *specSuite) TestSpecJitterSeed() {
```

Comment:
To me, this test is not very readable. The `checkSequenceFull` method name is pretty opaque. I can't really tell what this is testing.

Reply:
It's testing a sequence of getNextTime calls (the full version including jitter, not rawNextTime that doesn't do jitter)